apps/demo/src/app/pages/view-transition/view-transition-page.component.less

```diff
@@ -0,0 +1,56 @@
+:host {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+}
+
+.photos {
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    gap: 1rem;
+    margin: 1rem 0;
+
+    .photo {
```

Comment:
I think we do not need nesting here, one level structure is easier to read in my opinion

Reply:
i guess i've fixed it, take a look please